plugins/com.google.cloud.tools.eclipse.test.core/plugin.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This empty plugin.xml should go (causes warnings in the IDE).  Adding an extension or extension point from the manifest editor will recreate it when necessary